### DISCUSSION TITLE
Issue with setup

### DISCUSSION THREAD

**Discussion by darechan** [unstructuredstudio/zubhub#569] [Q&A]
Hi I was trying to set up ZubHub on my local machine but i am getting this issue on my system. 
> This is the issue with the back end
![image](https://user-images.githubusercontent.com/93364240/219867153-319a346c-ce8c-4cf7-a744-1fbdc666504f.png)

> Issue with front end
![image](https://user-images.githubusercontent.com/93364240/219867194-39e18b61-f2e9-4f2c-bee3-0afd814e489e.png)

**Comment by darechan** [comment]
![image](https://user-images.githubusercontent.com/93364240/223205568-cd0e4c00-91a4-40d0-b263-6fc91d472522.png)

Can anyone help me resolve this?

**Reply by yaya-mamoudou** [reply]
There is no issue on this screenshot you shared @darechan . You can ignore those. No need to do the `npm audit fix --force.` 
You can proceed to run your frontend application using npm start. 
In my own case, I couldn't run it using Nodejs v18. I had to use `Node version Manager(NVM) `to install and use `nodejs v16 or bellow` before the development server was able to run.

**Comment by psalmuelle** [comment]
@darechan , I see that you are trying to run the backend tests. Seems there is an error in the test_user_is_not_active. It returned `status 200 `instead of `status 400`, which is in sense a "true error" because you are not logged in on [http://localhost:8000/](http://localhost:8000/). 
Your backend is running fine!
For the frontend issue, how long did it take before you use the `control + C` command? Did you try to see if your PC is connected to the internet at that particular time? 
You can try again, if it still persists, follow @yaya-mamoudou's instruction.
